README.md

```diff
@@ -62,6 +62,12 @@ A: Indeed there is. See the example hooks.py file.
 Q: What about badly-formed HTML (long lines, etc..)?
 A: Use python-utidylib. See the example hooks.py file.
 
+Q: Is there a way to compare the data using a different diff tool (eg. syntax-aware XML diff)?
+A: Use "--diff-command=COMMAND", where COMMAND is a full filesystem path to the
+   tool followed by its arguments. Substitute compared files' names with '{OLD_DATA}'
+   and '{NEW_DATA}' tokens. This doesn't support pipelining of multiple
+   commands - put these in a sh script and use this instead.
```

Comment:
Given that we don't support pipelining (should we? subprocess has `shell=True` for that), maybe we shouldn't request `{OLD_DATA}` and `{NEW_DATA}` but just pass in the first and second argument? (and if the diff utility needs different parameter ordering and/or additional commands, just let a shell script deal with those, and call the shell script with the two positional arguments)